### PR TITLE
Code generation experiments

### PR DESCRIPTION
**Pull request by LeshaKoss**

### CODE REVIEW

File: scripts/_lib/list_locales.js [hunk 1]
```diff
@@ -6,7 +6,21 @@ export default function listLocales () {
...
     .filter((locale) => /^[^._]/.test(locale))
     .map((locale) => ({
-      name: locale,
+      name: camelize(locale),
+      path: `./locale/${locale}`,
       fullPath: `./src/locale/${locale}/index.js`
     }))
 }
+
+function camelize (str) {
```
Comment (havenchyk) [inline scripts/_lib/list_locales.js]: would you mind to use `lodash` for such cases and not to reinvent the wheel?

Comment (LeshaKoss) [inline scripts/_lib/list_locales.js]: @havenchyk `_.camelCase` would not suit here (I wanted to transform `zh_cn` into `zhCN` not `zhCn`)

Comment (havenchyk) [inline scripts/_lib/list_locales.js]: @LeshaKoss I see, `upperFirst(camelCase(str))` will solve your problem, but in this case maybe your solution is better

**Comment by LeshaKoss** [comment]
#368

**Comment by LeshaKoss** [comment]
(there were more changes in this PR but #355 made them needless)

**Pull request closed without merging**